### PR TITLE
#157731614 Add Dropset Support

### PR DESCRIPTION
**Pull request by dondrzzy**
### What does this PR do?
This PR creates a feature that enables users to add dropset workouts.

Add `changeWorkoutTyp`e function to `wger/core/static/js/wger-core.js`
Add a onClick event handler to function to `addEntry button` in `wger/manager/templates/day/log.html`
Add functionality to enable adding new sets to the existing sets in `wger/manager/views/log.py`
Create new model WorkoutType, add category field to Setting model as a
foreign key.

#### Description of Task to be completed?
Add support for dropsets((https://en.wikipedia.org/wiki/Drop_set)

#### How should this be manually tested?
1: Clone the application and install the requirements in your virtual environment.
2: Update the database settings in the `settings.py` file with your postgres user, password and database name.
3: Run the application with `python manage.py runserver`.
4: Register and login.
5: Hover over `Workouts` in the navbar menu and select `Workouts`.
6: Add a `workout` then a `day` and `description` then add an `exercise` to the  workout: 
7: Add an initial weight and amount
8: You should be able to declare a workout type as a `dropset` or `default` workout.

##### Expected response
screen shot 2018-06-17 at 16 37 30

8: Click save and you should view the created workout as a dropset workout.

##### Expected response
screen shot 2018-06-17 at 16 37 57

9: Go to `Workouts`-> `carlendar` and add new log to the workout of the day respectively.
10: You should be able to add a new dropset reports while you work out.

##### Expected response
screen shot 2018-06-17 at 16 38 41

#### Any background context you want to provide?
This feature was not available.

#### What are the relevant pivotal tracker stories?
#157731614

### CODE REVIEW

**Comment by peterwade153** [comment]
work on the failing tests

File: wger/core/static/js/wger-core.js [hunk 1]
```diff
@@ -452,6 +466,7 @@ function updateAllExerciseFormset() {
             $formsets.append(data);
             $('#exercise-search-log').scrollTop(0);
             $formsets.trigger('create');
+            console.log('value: ', setValue);
```
Comment (emugaya) [inline wger/core/static/js/wger-core.js]: Remove this statement

File: wger/core/templates/tags/render_day.html [hunk 2]
```diff
@@ -8,33 +8,33 @@
     wgerSetupSortable();
 
     // Init the modal dialog to edit the exercise
-    wgerFormModalDialog();
+    // wgerFormModalDialog();
```
Comment (emugaya) [inline wger/core/templates/tags/render_day.html]: Why are you commenting this line?

File: wger/manager/fixtures/categories.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        }
+    }, 
+    {
+        "pk": 2, 
+        "model": "manager.WorkoutType",
+        "fields": {
+            "category": "dropset" 
+        }
+    }
+]
```
Comment (emugaya) [inline wger/manager/fixtures/categories.json]: Add new line

**Review by emugaya — CHANGES_REQUESTED** [review]
Do remove the multiple migration files you have. We need to have one migration per `django app`

**Comment by emugaya** [comment]
Change the `dropset` from a dropdown to a checkbox. and It should be unselected by default. FIx your tests..

**Comment by hariclerry** [comment]
@dondrzzy, awesome job. Please don't forget to run autopep8 command to fix all pep8 issues in the new lines of code you added. Coveralls is still complaining though.

**Pull request merged**